app/controllers/api/v1/custom_emojis_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class Api::V1::CustomEmojisController < Api::BaseController
   vary_by '', unless: :disallow_unauthenticated_api_access?
+  skip_before_action :require_authenticated_user!, unless: :whitelist_mode?
```

Comment:
Isn't this also a catstodon specific thing to allow anyone to copy custom emojis?

Reply:
It is and should not be in here.